client/tests/integration/unstable_network.rs

```diff
@@ -58,7 +58,7 @@ fn unstable_network(
     polling_max_attempts: u32,
     polling_period: Duration,
 ) {
-    drop(iroha_logger::install_panic_hook());
+    let _install_hook_result = iroha_logger::install_panic_hook();
```

Comment:
```suggestion
    if let Err(err) = iroha_logger::install_panic_hook() {
        eprintln!("Installing panic hook failed: {err}");
    }   
```

Reply:
Then you're not ignoring the error anymore. Could also add a helpful message on how to avoid it, or what might cause it.